src/components/organisms/Notifications.tsx

```diff
@@ -0,0 +1,264 @@
+    id: 19,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 20,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 21,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 22,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 23,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 24,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 25,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 26,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 27,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+  {
+    id: 28,
+    date: "01/01/22 14:22",
+    content: "messagemessagemessagemessagemessage",
+    is_read: false,
+  },
+];
+
+// ToDo -> implement when message click
+const handleClickMessage = (data: any) => {};
+
+export default function Notifications() {
+  const [readState, setReadState] = useState<string>("all");
+  const [notifiData, setNotifiData] = useState<Array<NotificationsData> | undefined>([]);
+
+  useEffect(() => {
+    setNotifiData(allNotifiData);
+  }, []);
+
+  const handleClickState = (state: string) => {
+    setReadState(state);
+    switch (state) {
+      case "all":
+        setNotifiData(allNotifiData);
+        break;
+
+      case "unread":
+        const unreadData = allNotifiData.filter((elm) => !elm.is_read);
+        setNotifiData(unreadData);
+        break;
+
+      case "read":
+        const readData = allNotifiData.filter((elm) => elm.is_read);
+        setNotifiData(readData);
+        break;
+    }
+  };
+
+  return (
+    <div className={styles.notifi}>
+      <div className={styles.notifi__inner}>
+        <nav className={styles.notifi__menu}>
+          <button
+            className={`${styles.notifi__menuButton} ${
+              readState === "all" ? styles.notifi__read : ""
+            }`}
+            onClick={() => handleClickState("all")}
+          >
+            All
+          </button>
+          <button
+            className={`${styles.notifi__menuButton} ${
+              readState === "unread" ? styles.notifi__read : ""
+            }`}
+            onClick={() => handleClickState("unread")}
+          >
+            Unread
+          </button>
+          <button
+            className={`${styles.notifi__menuButton} ${
+              readState === "read" ? styles.notifi__read : ""
+            }`}
+            onClick={() => handleClickState("read")}
+          >
```

Comment:
Change the each state condition to `readState === "all"` to `readState === ReadState.All`.